example.env

```diff
@@ -1,2 +1,3 @@
 DJANGO_DEBUG=True
 DJANGO_SECRET_KEY="DJANGO_SECRET_KEY"
+NPM_BIN_PATH = "C:\\Program Files\\nodejs\\npm.cmd"
```

Comment:
Not a fan of polluting Django with references to node, but we'll deal with that later (probably)